### ISSUE TITLE
WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.

### ISSUE THREAD

**Issue by armab** [StackStorm/st2-packages#516]
When running StackStorm install (`st2mistral` DB migration script), a big wall of `WARNING` is showed up.

This is confusing from the user's perspective and was already mentioned in #community:

```
20171025T160122+0000 INFO  [alembic.runtime.migration] Running upgrade 020 -> 021, Increase environments_v2 column size from JsonDictType to JsonLongDictType
20171025T160122+0000 INFO  [alembic.runtime.migration] Running upgrade 021 -> 022, namespace_support
20171025T160122+0000 INFO  [alembic.runtime.migration] Running upgrade 022 -> 023, Add the root execution ID to the workflow execution model
20171025T160122+0000 INFO  [alembic.runtime.migration] Running upgrade 023 -> 024, Add database indices
20171025T160126+0000 2017-10-25 16:01:24.555 9626 INFO mistral.db.sqlalchemy.migration.cli [-] populating db
20171025T160126+0000 2017-10-25 16:01:26.733 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.734 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.734 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.734 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.734 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.735 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.735 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.736 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.736 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.736 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.736 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.737 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.737 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.737 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.738 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.738 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.738 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.738 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.739 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.739 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.739 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.740 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.740 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.740 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.741 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.741 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.741 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160126+0000 2017-10-25 16:01:26.742 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.742 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.742 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.743 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.743 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.743 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.744 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.744 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.744 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.745 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.745 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.745 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.746 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.746 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.746 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.747 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.748 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.749 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.749 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.749 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.750 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.750 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.750 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.751 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.751 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.751 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.752 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 2017-10-25 16:01:26.752 9626 WARNING mistralclient.auth.keystone [-] You must either provide a valid token or a password (api_key) and a user.
20171025T160127+0000 Starting mistral-api:                 [  OK  ]
20171025T160128+0000 Starting mistral-server:              [  OK  ]
```

Things like this in installer will always bring user's attention and typically will be raised again and again, wasting #community time.
These issues are minor, but should be fixed similar to https://github.com/StackStorm/st2-packages/issues/490

**Comment by punkrokk** [comment]
@armab Would these need to be addressed on v3.3 if we are deprecating Mistral?

**Comment by armab** [comment]
@punkrokk This is assigned to `v3.3.0` Project as with the Mistral deprecation we'll have a chance to close this issue.

FYI @StackStorm/tsc  We'll need to do some housecleaning work once we deprecate both `EL6` and `Mistral` in `v3.3.0` as there are so many issues/bugs/reports/questions to close.
